src/redux/reducers/user.ts

```diff
@@ -127,16 +130,18 @@ const userSlice = createSlice({
     });
     builder.addCase(registerUser.fulfilled, (store, { payload }) => {
       store.error = undefined;
-      return {
-        ...store,
+      localStorage.setItem("accessToken", payload.accessToken);
+      localStorage.setItem("refreshToken", payload.refreshToken);
+      Object.assign(store, {
+        ...userInitialState,
         ...payload.user,
```

Comment:
Better to use return with spread approach, instead of Object.assign

Reply:
spread did not save user to store